test/WebJobsStartupTests/WebJobsStartupTests.csproj

```diff
@@ -21,7 +21,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.20" />
-    <PackageReference Include="Microsoft.Net.Sdk.Functions" Version="3.0.*" />
+    <PackageReference Include="Microsoft.Net.Sdk.Functions" Version="3.0.9" />
```

Comment:
Was this an accidental change or deliberate?

Reply:
I undid the change. For whatever reason, when I compile locally I get the package restore error below. If the CI passes, it might be some issue on my machine.

Retrying 'FindPackagesByIdAsync' for source 'https://msblox.pkgs.visualstudio.com/_packaging/de9af483-3d68-4e7c-88a3-a79624abdade/nuget/v3/flat2/microsoft.net.sdk.functions/index.json'.
3>A task was canceled.
3>Retrying 'FindPackagesByIdAsync' for source 'https://msblox.pkgs.visualstudio.com/_packaging/de9af483-3d68-4e7c-88a3-a79624abdade/nuget/v3/flat2/microsoft.net.sdk.functions/index.json'.
3>A task was canceled.